service-oriented-architecture/src/main/java/com/iluwatar/soa/controller/GreetingController.java

```diff
@@ -0,0 +1,21 @@
+package com.iluwatar.soa.controller;
+
+import com.iluwatar.soa.services.interfaces.PersonalizedGreetingService;
+import com.iluwatar.soa.services.registry.ServiceRegistry;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("home")
+public class GreetingController {
```

Comment:
Maybe we can consider a @ControllerAdvice to handle errors in controllers? or we can handle it try / catch if something goes wrong?